### ISSUE TITLE
envoy main there is no configured limit to the number of allowed active connections. Set a limit via the runtime key overload.global_downstream_max_connections

### ISSUE THREAD

**Issue by kevin-shelaga** [istio/istio#29844]
The following error is thrown every second or so with a minimal istio install. I am not sure how to resolve.

```
envoy main	there is no configured limit to the number of allowed active connections. Set a limit via the runtime key overload.global_downstream_max_connections
```

Is there a way to disable this log without specifying overload.global_downstream_max_connections, or how can I set this value to its max allowed value?

**Comment by hzxuzhonghu** [comment]
>The following error is thrown every second or so with a minimal istio install

IMO, envoy should have a fix on this

**Comment by ramaraochavali** [comment]
> IMO, envoy should have a fix on this

What fix? defaulting to max int? I think it was left like that to force control planes choose the relevant value.

**Comment by hzxuzhonghu** [comment]
I mean envoy should not throw an error every second.
I am not sure why envoy suggests configuring the value to half the ulimit of the system.

**Comment by ramaraochavali** [comment]
@PiotrSikora might know about the reason for Envoy suggestion

**Comment by JCMDuarte** [comment]
https://www.envoyproxy.io/docs/envoy/latest/configuration/operations/overload_manager/overload_manager#limiting-active-connections

**Comment by johnzheng1975** [comment]
In istio1.22.4, still show such alert:
![image](https://github.com/user-attachments/assets/2a88ed7d-e0d3-46bd-a699-e12c3a297e15)

Is it possible to remove such log by default? The log level is warning, it make user think our configure is incorrect.
Does istio has default value for this? 

Thanks.

**Comment by johnzheng1975** [comment]
Seems this issue is resolved in new version:
https://github.com/envoyproxy/envoy/pull/30620
https://github.com/istio/istio/issues/49490

**Comment by sgrissom-rxss** [comment]
@johnzheng1975 for clarification, _which_ version of Istio is this fixed in? From what I can tell, if https://github.com/envoyproxy/envoy/pull/30620 fixes it, it should have been available since envoy v1.29, yeah? According to the [Istio/Envoy relationship table](https://istio.io/latest/docs/releases/supported-releases/) istio 1.23 should have Envoy v1.31, but I see this same message repeatedly using Istio 1.23.3. Feel like I'm missing something silly..
